ontology/oostt.md

```diff
@@ -18,6 +18,19 @@ products:
 - id: oostt.owl
 repository: https://github.com/OOSTT/OOSTT
 tracker: https://github.com/OOSTT/OOSTT/issues
+usages:
+- description: OOSTT terms are used in the CAFE questionnaire to assess trauma centers and systems.
```

Comment:
Since there is no way to inspect the codes that are in use, could you enumerate some of the terms that are used in the CAFE Questionnaire? 

For example: "OOSTT terms, such as OSTT:nnnnnnnn 'term label', are used in the CAFE questionnaire to assess trauma centers and systems."